.github/workflows/configs/test-integration-rosa-matrix.yaml

```diff
@@ -0,0 +1,29 @@
+matrix:
+    distro:
+      - name: OpenShift 4.15
+        type: openshift
+        # renovate: datasource=endoflife-date depName=red-hat-openshift versioning=regex:^4(\.(?<minor>\d+))?(\.(?<patch>\d+))?$
+        # /!\ Warning: When a new minor version of OpenShift is released, you must update all N-1, N-2, and N-3 versions in this matrix.
+        # rationale: Red Hat supports the last four minor versions of OpenShift. Therefore, to ensure compatibility and support, we must test against these versions. For more details, refer to the official support policy at https://endoflife.date/red-hat-openshift.          version: 4.15.13
+        version: 4.15.11
+        platform: rosa   
+
+      - name: OpenShift 4.14
+        type: openshift
+        # renovate: datasource=endoflife-date depName=red-hat-openshift versioning=regex:^4(\.(\d+))?(\.(?<patch>\d+))?$
+        version: 4.14.24
+        platform: rosa
+
+      # # TODO: in May 2024, ROSA does not supports OpenShift 4.13 with HCP, when this will be == 4.14,  un-comment this block
+      # - name: OpenShift 4.13
+      #   type: openshift
+      #   # renovate: datasource=endoflife-date depName=red-hat-openshift versioning=regex:^4(\.(\d+))?(\.(?<patch>\d+))?$
+      #   version: 4.13.41
+      #   platform: rosa
+
+      # # TODO: in May 2024, ROSA does not supports OpenShift 4.12 with HCP, when this will be == 4.14,  un-comment this block
+      # - name: OpenShift 4.12
+      #   type: openshift
+      #   # renovate: datasource=endoflife-date depName=red-hat-openshift versioning=regex:^4(\.(\d+))?(\.(?<patch>\d+))?$
+      #   version: 4.12.56
+      #   platform: rosa
```

Comment:
since < 4.14 isn't supported.
Maybe just a general generic example on how it should look like since one can not reuse this for 4.12 or 4.13?
Or will they support it in the future, not sure what I'm expected to do when OpenShift hits 4.14 🤷